src/babel/index.js

```diff
@@ -962,7 +962,13 @@ function resolveRNStylesheet(t, platform, target, stylesheet) {
         resolvedProperties.push(property)
       }
       const pushFromInner = objectExpr => {
-        objectExpr.properties.forEach(pushProp)
+        objectExpr.properties.forEach(property => {
+          if (property.type === 'ObjectProperty' && property.key.name === '_mixin' && property.value.type === 'ObjectExpression') {
```

Comment:
Do you think we will ever support composing multiple mixins?

Reply:
I don't think so. For mixins to be able to work with CSS, we need them to be injected by a Babel plugin supplying a literal object. Hence, there is no reason for that object to be composed with other `_mixin`'s and not just computed by composing with `...`